packages/editor/src/services/list-util/markdown-list-util.ts

```diff
@@ -0,0 +1,46 @@
+import { EditorView } from '@codemirror/view';
```

Comment:
型情報のみを使うのであれば import type を使う
```suggestion
import { type EditorView } from '@codemirror/view';
```